docs/cosmos-hub/gaiacli.md

```diff
@@ -524,37 +524,87 @@ You can also query all of the delegations to a particular validator:
 
 ### Governance
 
-Governance is the process from which users in the Cosmos Hub can come to consensus on software upgrades, parameters of the mainnet or on custom text proposals. This is done through voting on proposals, which will be submitted by `Atom` holders on the mainnet.
+Governance is the process from which users in the Cosmos Hub can come to consensus
+on software upgrades, parameters of the mainnet or signaling mechanisms through
+text proposals. This is done through voting on proposals, which will be submitted
+by `ATOM` holders on the mainnet.
```

Comment:
I don't think we really have to capitalize it everywhere

Reply:
Agreed. I can't recall if this was a Github suggestion or already exist in the docs.